src/sidebar/util/time.ts

```diff
@@ -262,3 +262,21 @@ export function formatDate(date: Date, Intl?: IntlType): string {
     Intl,
   );
 }
+
+/**
+ * Formats a date as `YYYY-MM-DD hh:mm`, using 24h and system timezone.
+ */
+export function formatDateTime(date: Date): string {
+  // First we convert the date to system timezone.
```

Comment:
Dates don't have a timezone - they are always represented as "count of milliseconds since Jan 1st 1970 UTC" internally. The timezone conversion happens when you call `Date.getMonth` etc. You can simplify this to just use `date` everywhere.

Reply:
You are right. This is what I meant when I said I always find timezone-related stuff confusing 😅 

Also, I now better understand what you meant by how dates are persisted.